src/redux/slices/editingV2Project.ts

```diff
@@ -0,0 +1,176 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import * as constants from '@ethersproject/constants'
+import { ProjectMetadataV3 } from 'models/project-metadata'
+import { V2FundAccessConstraint } from 'models/v2/fundingCycle'
+
+import {
+  ETHPayoutGroupedSplits,
+  ReserveTokenGroupedSplits,
+  Split,
+} from 'models/v2/splits'
+import {
+  serializeV2FundingCycleData,
+  serializeV2FundingCycleMetadata,
+  SerializedV2FundingCycleData,
+  SerializedV2FundingCycleMetadata,
+  SerializedV2FundAccessConstraint,
+} from 'utils/v2/serializers'
+import { parsePerbicent, parsePermille } from 'utils/formatNumber'
+
+import {
+  ETH_PAYOUT_SPLIT_GROUP,
+  RESERVE_TOKEN_SPLIT_GROUP,
+} from 'constants/v2/splits'
+import { DEFAULT_BALLOT_STRATEGY } from 'constants/ballotStrategies/ballotStrategies'
+
+export interface EditingV2ProjectState {
+  version: number
+  projectMetadata?: ProjectMetadataV3
+  fundingCycleData?: SerializedV2FundingCycleData
+  fundingCycleMetadata?: SerializedV2FundingCycleMetadata
+  fundAccessConstraints?: V2FundAccessConstraint[]
+  payoutSplits?: ETHPayoutGroupedSplits
+  reserveTokenSplits?: ReserveTokenGroupedSplits
+}
+
+export interface V2ProjectState {
```

Comment:
what's the difference between how this and `EditingV2ProjectState` are used?

Reply:
Ah great catch! Looks like `EditingV2ProjectState` isn't actually used anymore 🙈 . Removed!